std/builtin/logic.neva

```diff
@@ -0,0 +1,26 @@
+#extern(match)
+pub component Match<T>(data T, [case] T) ([case] T, else T)
+
+#extern(eq)
+pub component Eq<T>(actual T, expected T) (res bool)
+
+#extern(if)
+pub component If(data bool) (then any, else any)
+
+#extern(not)
+pub component Not(data bool) (res bool)
+
+#extern(and)
+pub component And(A bool, B bool) (res bool)
+
+#extern(or)
+pub component Or(A bool, B bool) (res bool)
+
+#extern(int int_is_greater, float float_is_greater)
+pub component IsGreater<T int | float>(actual T, compared T) (res bool)
+
+#extern(int_is_even)
+pub component IsEven(data int) (res bool)
```

Comment:
1. I don't think we need this in stdlib
2. It's possible to implement it in pure neva, no need for runtime function

Reply:
Closing -> Rempved IsEven